### PR TITLE
Employer open position - frontend integration

### PR DESCRIPTION
**Pull request by ilhoKim**
# Description

- Employer Open position add page UI with link from sign up
- Backend hookup required
- UI touch-up required

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Self review

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: frontend/lambda-in/src/Components/OpenPositionAdd/OpenPositionAdd.js [hunk 1]
```diff
@@ -1,165 +1,183 @@
...
-   */
+  constructor(props) {
+    super(props);
+    this.state = {
+      projectName: "",
+      description: "",
+      jobTitle: "",
+      techStack: "",
+      skills: "",
+      minSalrary: "",
```
Comment (d-vail) [inline frontend/lambda-in/src/Components/OpenPositionAdd/OpenPositionAdd.js]: Typo

File: frontend/lambda-in/src/Components/OpenPositionAdd/OpenPositionAdd.js [hunk 2]
```diff
@@ -1,165 +1,183 @@
...
+      maxSalary: ""
```
Comment (d-vail) [inline frontend/lambda-in/src/Components/OpenPositionAdd/OpenPositionAdd.js]: Typo

Comment (ilhoKim) [reply frontend/lambda-in/src/Components/OpenPositionAdd/OpenPositionAdd.js]: Thank you! Typo fixed.

**Comment by ilhoKim** [comment]
Thanks Danny! Typo fixed.

**Pull request merged**